src/reference/informative/informA2_6.rst

```diff
@@ -0,0 +1,97 @@
+.. _inform2_6:
+
+.. container:: toggle
+
+  .. container:: header
+
+    See more
+
+  .. container:: infospec
+
+    .. container:: heading3
+
+      Understanding element IDs
+
+    In order to understand how the XML :code:`id` attribute is used in CellML models, we need to remember three things.
+    Firstly, in CellML the elements are identified by uniqueness over the *model* scope of their :code:`name` attribute, their type (eg: :code:`variable`), and in some cases, by their parent element too.
+    Secondly, the CellML :code:`model` might exist over several different *documents*, and make use of :code:`import` functionality to connect these documents into one *model*.
+    Finally, the uniqueness of the XML :code:`id` attribute value extends only to the *document* scope, not the CellML *model* scope.
+    Consider the example below.
+
+    .. code::
+
+      # In the document "BreakfastMenu.cellml".
+      # All id attributes are unique within this document.
+      model: name = "BreakfastOfChampions", id = "b"
+        ├─ component: name = "Protien", id = "p"
+        ├─ component: name = "Carbohydrates", id = "c" <╴╴╴╴╴╴╴╴╴┐
+        └─ component: name = "FunStuff", id = "f"                ╷
+             └─ variable: name = "FunStuff", id = "fs"           ╷
+                                                                 ╷
+                                              Valid: Component named "Porridge" is
+                                              imported, but its id attribute "p" is not.
+                                                                 ╵
+      # In the document "CarbohydrateIdeas.cellml".              ╵
+      # All id attributes should be unique within this document. ╵
+      model: name = "Carbs" id = none specified                  ╵
+        ├─ component: name = "Chocolate", id = "c"               ╵
+        ├─ component: name = "Porridge", id = "p" ╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴┘
+        └─ component: name = "Toast", id = "t"
+            └─ variable: name = "Peanutbutter", id = "p" 
+
+            # Invalid: The variable Peanutbutter is not allowed to have an id of "p"
+            # as this conflicts with the component Porridge id in the same document.
+
+
+    .. container:: toggle
+
+      .. container:: header
+
+        See CellML syntax
```

Comment:
Alan's going to want a full stop here, isn't he

Reply:
He's not gonna get one :) It's more of a button/heading than a complete sentence.